src/Poseidon/servertypes.txt

```diff
@@ -43,8 +43,7 @@ charListPacket 0x99d
 maploginPacket 022D
 charBlockSize 155
 
-[vRO]
-decrypt_mid 1
+[vRO_2021-07-13]

```

Comment:
I think we must use 2021-07-08 (date from ragexe.exe)

Reply:
in bRO we used to get data from .gpf / .rgz file, users was redirected to ragnarok ftp page and because of this name was more easy to find the right one

Reply:
what about
http://dl.rag.vtcgame.vn/Live/patchfile/2021-07-07_Ragexe_1625744411.rgz

Reply:
ok, i just confused the file names

Reply:
fixed at https://github.com/OpenKore/openkore/pull/3460/commits/e752f2e5d4966b00385bf89d1dbbe0a3c3162cee